src/main/resources/assets/iris/lang/es_es.json

```
@@ -0,0 +1,6 @@
+{
+  "iris.shaders.reloaded": "Shaders actualizados!",
+  "iris.keybind.reload": "Actualizar los Shaders",
+  "iris.keybinds": "Iris",
+  "iris.shaders.reloaded.failure": "Fallo en la actualización de los shaders: %s"
```

I think the correct word in Spanish for "reload" would be "recargar", hence the following suggestion:
```suggestion
  "iris.shaders.reloaded": "Shaders recargados!",
  "iris.keybind.reload": "Recargar Shaders",
  "iris.keybinds": "Iris",
  "iris.shaders.reloaded.failure": "Error al recargar los shaders: %s"
```
I'd say "actualizar" is more like "update".